packages/rspack/src/watching.ts

```diff
@@ -238,15 +238,18 @@ class Watching {
 		this.compiler.hooks.watchRun.callAsync(this.compiler, err => {
 			if (err) return this._done(err, null);
 
-			const isRebuild = this.compiler.options.devServer && !this.#initial;
+			const canRebuild =
+				this.compiler.options.devServer &&
```

Comment:
What happens when people use Rspack via nodejs instead of `@rspack/cli`?

Reply:
No `options.devserver` won't run rebuild, I'll fix that in another PR.